docs/extensions.rst

```diff
@@ -0,0 +1,60 @@
+.. _extensions:
+
+##########
+Extensions
+##########
+
+Specialized extensions are provided for specific use cases.
+
+
+*****************
+typing.NamedTuple
+*****************
+
+If you are working with :class:`typing.NamedTuple` definitions, you may use :class:`~marshmallow_annotations.ext.namedtuple.NamedTupleSchema`
+to generate your schema. This modifies loading behavior to deserialize
+directly to instances of your defined :class:`~typing.NamedTuple`::
+
+    from marshmallow_annotations import NamedTupleSchema
+    from typing import NamedTuple, Optional
+
+    class Vector(NamedTuple):
+        x: int
+        y: Optional[int]
+        z: Optional[int] = 5
+
+    class VectorSchema(NamedTupleSchema):
+        class Meta:
+            target = Vector
+
+    schema = VectorSchema()
+    schema.load({'x': 1}).data
+
+    # Vector(x=1, y=None, z=5)
+
+    schema.dump(Vector(x=1, y=None, z=5)).data
+
+    # {'x': 1, 'y': None, 'z': 5}
+
+
+Additionally, the ``Meta`` class provides you with the option flag
+``dump_default_args`` to control whether attribute values matching defaults
+should be dumped or ignored::
```

Comment:
- should be dumped or ignored::
    + should be dumped or ignored, the default is to dump them::

Additionally, and I won't block on this, I'm a big fan of using two-value enums rather than booleans when possible:

    class DefaultArgBehavior(Enum):
        dump = auto()
        ignore = auto()

And using `default_args = DefaultArgBehavior.dump`

But that's purely personal preference.